spec/system/bus_stops/managing_bus_stops_spec.rb

```diff
@@ -0,0 +1,63 @@
+require 'spec_helper'
+
+describe 'managing stops as an admin' do
+  let(:admin) { create :user, :admin }
+  let!(:bus_stop) { create :bus_stop }
+  before :each do
+    when_current_user_is admin
+    visit manage_bus_stops_url
+  end
+  context 'delete button' do
+    it 'deletes the specific bus stop' do
+      expect(page).to have_selector 'table.manage tbody tr', count: 1
+      within 'tr', text: bus_stop.name do
+        click_button 'Delete'
+      end
+      page.driver.browser.switch_to.alert.accept
```

Comment:
You could set `page.driver.browser.switch_to.alert` to a variable to make it's value more clear.

Reply:
@dfaulken mentioned something similar, but said that I didn't need to unless the line appeared multiple times. But if you think it's absolutely necessary I will make the appropriate changes.